package.json

```diff
@@ -6,9 +6,29 @@
   "main": "src/app/index.html",
   "repository": "https://github.com/Netflix/vector",
   "license": "Apache-2.0",
-  "dependencies": {},
+  "dependencies": {
+    "angular": "~1.4.10",
+    "angular-mocks": "~1.4.10",
+    "angular-route": "~1.4.10",
+    "angular-sanitize": "~1.4.10",
+    "angular-toastr": "~1.5.0",
+    "angular-ui-bootstrap": "^2.5.6",
+    "angular-ui-sortable": "^0.19.0",
+    "animate.css": "~3.4.0",
+    "bootstrap": "~3.3.6",
+    "d3": "3.5.16",
+    "font-awesome": "4.6.1",
+    "jquery": "~2.1.4",
+    "lodash": "~4.5.1",
+    "malhar-angular-dashboard": "git+https://github.com/dtpublic/malhar-angular-dashboard.git",
+    "moment": "~2.10.6",
+    "nvd3": "1.8.2"
+  },
   "scripts": {
-    "test": "gulp test"
+    "clean": "gulp clean",
+    "build": "gulp build",
+    "test": "gulp test",
+    "serve": "gulp serve"
   },
   "devDependencies": {
```

Comment:
Missing `editorconfig`.

Reply:
TIL editorconfig

Reply:
@spiermar is `editorconfig` actually needed as a dep of this project? I don't see why editorconfig would need to be part of the project dev deps.

Reply:
as discussed, will leave this out and do linter cleanups as a separate pr